api/src/main/java/org/openmrs/PatientIdentifier.java

```diff
@@ -328,4 +331,23 @@ public int compare(PatientIdentifier pi1, PatientIdentifier pi2) {
 			return retValue;
 		}
 	}
+
+
+	/**
+	 * This method returns a patient program associated to the identifier in context
```

Comment:
We tend to avoid what is already obvious. For instance, everyone knows that this is a method. :)